model/otlpgrpc/metrics.go

```diff
@@ -36,6 +36,24 @@ func NewMetricsResponse() MetricsResponse {
 	return MetricsResponse{orig: &otlpcollectormetrics.ExportMetricsServiceResponse{}}
 }
 
+// MetricsRequest represents the response for gRPC client/server.
+type MetricsRequest struct {
+	orig *otlpcollectormetrics.ExportMetricsServiceRequest
+}
+
+// NewMetricsRequest returns an empty MetricsRequest.
+func NewMetricsRequest() MetricsRequest {
```

Comment:
`NewMetricsRequestWithMetrics` method since that seems like a common use-case. The same question applies to traces/logs as well.

Reply:
Hmm.. I understand but would like to keep the API closer to pdata. Also half of the case will be removed when we will add marshal/unmarshal.